doc/source/ray-air/doc_code/checkpoint_mlflow.py

```diff
@@ -0,0 +1,31 @@
+# __mlflow_checkpoint_start__
+from ray.air.checkpoint import Checkpoint
+from sklearn.ensemble import RandomForestClassifier
+import mlflow.sklearn
+
+# Create an sklearn classifier
+clf = RandomForestClassifier(max_depth=7, random_state=0)
+# ... e.g. train model with clf.fit()
+# Save model using MLflow
+mlflow.sklearn.save_model(clf, "model_directory")
+
+# Create checkpoint object from path
+checkpoint = Checkpoint.from_directory("model_directory")
+
+# Convert into dictionary
+checkpoint_dict = checkpoint.to_dict()
+
+# This dict can then be passed around, e.g. to a different function
+
+# At some other location, recover checkpoint object from dict
+checkpoint = Checkpoint.from_dict(checkpoint_dict)
```

Comment:
Similar concern as above, in this example I don't see what the value of having a `Checkpoint` is.